ibc-derive/src/utils.rs

```diff
@@ -30,6 +30,16 @@ impl Imports {
         &self.prefix
     }
 
+    pub fn client_validation_context(&self) -> TokenStream {
+        let Prefix = self.prefix();
+        quote! {#Prefix::client::context::ClientValidationContext}
+    }
+
+    pub fn client_execution_context(&self) -> TokenStream {
+        let Prefix = self.prefix();
+        quote! {#Prefix::client::context::ClientExecutionContext}
+    }
+
```

Comment:
I think these two are used in those unused variables.

Reply:
[Likewise](https://github.com/cosmos/ibc-rs/pull/1127#discussion_r1541909049), we can discard these methods as well.

Reply:
b8a3ba8c